### ISSUE TITLE
grader.export() fails when no manually graded questions are in the notebook

### ISSUE THREAD

**Issue by taylorgibson** [ucbds-infra/otter-grader#403]
**Describe the bug**

When no manually graded questions are present in a notebook, grader.export() will produce an error and a .zip file that can not be submitted to gradescope. I believe this is because without any manually graded questions no output is sent to the latex --> PDF processing routine, resulting in an error from Latex.

Error message below for a notebook named practice.ipynb:
```
There was an error generating your LaTeX
Showing concise error message
============================================================
This is BibTeX, Version 0.99d (TeX Live 2019/Debian)
The top-level auxiliary file: notebook.aux
I found no \citation commands---while reading file notebook.aux
I found no \bibdata command---while reading file notebook.aux
I found no \bibstyle command---while reading file notebook.aux
(There were 3 error messages)

============================================================
---------------------------------------------------------------------------
FileNotFoundError                         Traceback (most recent call last)
/tmp/ipykernel_155/4245095150.py in <module>
      1 # Save your notebook first, then run this cell to export your submission.
----> 2 grader.export()

/opt/conda/lib/python3.9/site-packages/otter/check/utils.py in colab_only_method(self, *args, **kwargs)
     96         if self._colab:
     97             raise RuntimeError("This method is not compatible with Google Colab")
---> 98         return f(self, *args, **kwargs)
     99     return colab_only_method
    100 

/opt/conda/lib/python3.9/site-packages/otter/check/utils.py in run_function(self, *args, **kwargs)
    130                 except Exception as e:
    131                     self._log_event(event_type, success=False, error=e)
--> 132                     raise e
    133                 else:
    134                     self._log_event(event_type, results=results, question=question, shelve_env=shelve_env)

/opt/conda/lib/python3.9/site-packages/otter/check/utils.py in run_function(self, *args, **kwargs)
    125                         question, results, shelve_env = f(self, *args, **kwargs)
    126                     else:
--> 127                         results = f(self, *args, **kwargs)
    128                         shelve_env = {}
    129                         question = None

/opt/conda/lib/python3.9/site-packages/otter/check/notebook.py in export(self, nb_path, export_path, pdf, filtering, pagebreaks, files, display_link, force_save, run_tests)
    321             # convert(nb_path, filtering=filtering, filter_type=filter_type)
    322             export_notebook(nb_path, filtering=filtering, pagebreaks=pagebreaks)
--> 323             zf.write(pdf_path)
    324 
    325         if os.path.isfile(_OTTER_LOG_FILENAME):

/opt/conda/lib/python3.9/zipfile.py in write(self, filename, arcname, compress_type, compresslevel)
   1725             )
   1726 
-> 1727         zinfo = ZipInfo.from_file(filename, arcname,
   1728                                   strict_timestamps=self._strict_timestamps)
   1729 

/opt/conda/lib/python3.9/zipfile.py in from_file(cls, filename, arcname, strict_timestamps)
    499         if isinstance(filename, os.PathLike):
    500             filename = os.fspath(filename)
--> 501         st = os.stat(filename)
    502         isdir = stat.S_ISDIR(st.st_mode)
    503         mtime = time.localtime(st.st_mtime)

FileNotFoundError: [Errno 2] No such file or directory: 'practice.pdf'
```

**To Reproduce**
Steps to reproduce the behavior: run the `grader.export()` cell in a notebook that has no manually graded questions.

**Expected behavior**
In previous versions of otter, the template included some boilerplate text that would be sent over to Latex even if no manually graded questions were in the notebook, this may have prevented such an error.

**Versions**
Please provide your Python and Otter versions. The Otter can be obtained using `from otter import __version__`

Python version: 3.9.6
Otter-Grader version: 3.1.3

**Comment by chrispyles** [comment]
I think that this is probably not a change that we want to make, since these errors can be informative about cases in which a student accidentally changes something in the notebook that's needed for it to export correctly. What I'd recommend for cases in which you don't have any manually-graded questions is to use `grader.export(pdf=False)` so that it doesn't attempt to generate a PDF at all. You can set this up with Otter Assign using

```
export_cell:
    pdf: false
```

**Comment by taylorgibson** [comment]
Would it be possible to leave the error message, but find a way so that the zip file is still able to be submitted? Right now, after the error message is shown the zip that's generated can't be turned into Gradescope (says it's improperly formatted).

Or, how about an error message / warning in the `otter assign` process that's shown when there are no manually assigned questions, but the config would result in a PDF being generated? Right now there's not a mechanism to catch that a notebook would create this error until notebooks are already distributed to students and they are attempting to export them. Having an error in the `otter assign` command to catch it would allow the creator of the notebooks to have a way to catch this type of issue ahead of time.

**Comment by chrispyles** [comment]
Implemented a fix in #408
